dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/gains_network/gains_network_base_perpetual_trades.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    schema = 'gains_network_perpetual_trades',
+    alias = 'perpetual_trades',
+    post_hook='{{ expose_spells(\'["base"]\',
+                                    "project",
+                                    "gains_network",
+                                    \'["princi"]\') }}'
+        )
+}}
+
+{% set gains_network_base_perpetual_trade_models = [
+    ref('gains_network_v1_base_perpetual_trades')
+] %}
+
+SELECT *
+FROM
+(
+    {% for gains_network_perpetual_trades in gains_network_base_perpetual_trade_models %}
+    SELECT
+		blockchain
+		,block_date
+        ,block_month
+        ,block_time
```

Comment:
Please be consistent in `space/tab` indentation.
```suggestion
        blockchain
        ,block_date
        ,block_month
        ,block_time
```

Reply:
Maybe you can fix the tabs here.